include/z64save.h

```diff
@@ -852,117 +885,144 @@ typedef enum LinkAge {
 #define INFTABLE_197 0x197
 #define INFTABLE_198 0x198
 
-// 0x199-0x19F
-#define INFTABLE_199_19A_19B_19C_19D_19E_19F_INDEX 25
-#define INFTABLE_199_MASK (1 << 9)
-#define INFTABLE_19A_MASK (1 << 10)
-#define INFTABLE_19B_MASK (1 << 11)
-#define INFTABLE_19C_MASK (1 << 12)
-#define INFTABLE_19D_MASK (1 << 13)
-#define INFTABLE_19E_MASK (1 << 14)
-#define INFTABLE_19F_MASK (1 << 15)
-
-// 0x1A0-0x1AF
-#define INFTABLE_1AX_INDEX 26
-#define INFTABLE_1A0_SHIFT 0
-#define INFTABLE_1A1_SHIFT 1
-#define INFTABLE_1A2_SHIFT 2
-#define INFTABLE_1A3_SHIFT 3
-#define INFTABLE_1A4_SHIFT 4
-#define INFTABLE_1A5_SHIFT 5
-#define INFTABLE_1A6_SHIFT 6
-#define INFTABLE_1A7_SHIFT 7
-#define INFTABLE_1A8_SHIFT 8
-#define INFTABLE_1A9_SHIFT 9
-#define INFTABLE_1AB_SHIFT 11
-#define INFTABLE_1AD_SHIFT 13
+// INFTABLE 0x199-0x19F
+#define INFTABLE_KAKARIKO_CUCCO_INDEX (INFTABLE_199 >> 4)
+#define INFTABLE_199 0x199
+#define INFTABLE_19A 0x19A
+#define INFTABLE_19B 0x19B
+#define INFTABLE_19C 0x19C
+#define INFTABLE_19D 0x19D
+#define INFTABLE_19E 0x19E
+#define INFTABLE_19F 0x19F
+
+#define INFTABLE_RESET_KAKARIKO_CUCCOS()                                                                              \
+    gSaveContext.save.info.infTable[INFTABLE_KAKARIKO_CUCCO_INDEX] &=                                                 \
+        (u16) ~(GET_INFTABLE_MASK(INFTABLE_199) | GET_INFTABLE_MASK(INFTABLE_19A) | GET_INFTABLE_MASK(INFTABLE_19B) | \
+                GET_INFTABLE_MASK(INFTABLE_19C) | GET_INFTABLE_MASK(INFTABLE_19D) | GET_INFTABLE_MASK(INFTABLE_19E) | \
+                GET_INFTABLE_MASK(INFTABLE_19F));
+
+
+// INFTABLE 0x1A0-0x1AF
+#define INFTABLE_OVERWORLD_ENTRANCE_ICON_INDEX (INFTABLE_1A0 >> 4)
```

Comment:
Idk. I can get behind `EVENTCHKINF_INDEX_*`, but `EVENTCHKINF_MASK_*` seems strange to do since the mask should be associated stronger to the flagId name, not EVENTCHKINF.